ui/src/components/executions/ExecutionRoot.vue

```diff
@@ -1,20 +1,33 @@
 <template>
-    <b-card no-body>
-        <b-tabs card>
-            <b-tab
-                v-for="tab in tabs"
-                :key="tab.tab"
-                @click="setTab(tab.tab)"
-                :active="$route.query.tab === tab.tab"
-                :title="tab.title"
-                lazy
-            >
-                <b-card-text>
-                    <div :is="tab.tab" @follow="follow" />
-                </b-card-text>
-            </b-tab>
-        </b-tabs>
-    </b-card>
+    <div>
+        <b-card no-body>
+            <b-tabs card>
+                <b-tab
+                    v-for="tab in tabs"
+                    :key="tab.tab"
+                    @click="setTab(tab.tab)"
+                    :active="$route.query.tab === tab.tab"
+                    :title="tab.title"
+                    lazy
+                >
+                    <b-card-text>
+                        <div :is="tab.tab" @follow="follow" />
+                    </b-card-text>
+                </b-tab>
+            </b-tabs>
+        </b-card>
+        <bottom-line>
+            <ul class="navbar-nav ml-auto">
+                <li class="nav-item">
+                    <router-link
+                        :to="{name:'flowEdit', params: {namespace:$route.params.namespace, id: $route.params.flowId}, query:{tab: 'execution-configuration'}}"
+                    >
+                        <b-button>{{ $t('Execute') }}</b-button>
```

Comment:
I would prefer a new word here: like `New execution` to understand that's a new execution and don't change the current one